contracts/flashloan/FlashLender.vy

```diff
@@ -0,0 +1,95 @@
+# @version 0.3.10
+"""
+@title crvUSD FlashLender
+@notice ERC3156 contract for crvUSD flash loans
+@author Curve.Fi
+@license Copyright (c) Curve.Fi, 2020-2024 - all rights reserved
+"""
+
+interface ERC20:
+    def transferFrom(_from: address, _to: address, _value: uint256) -> bool: nonpayable
+    def transfer(_to: address, _value: uint256) -> bool: nonpayable
+    def decimals() -> uint256: view
+    def approve(_spender: address, _value: uint256) -> bool: nonpayable
+    def balanceOf(_from: address) -> uint256: view
+
+interface Factory:
+    def stablecoin() -> address: view
+
+interface ERC3156FlashBorrower:
+    def onFlashLoan(initiator: address, token: address, amount: uint256, fee: uint256, data: Bytes[10**5]) -> bytes32: nonpayable
+
+
+CALLBACK_SUCCESS: public(constant(bytes32)) = keccak256("ERC3156FlashBorrower.onFlashLoan")
+supportedTokens: public(HashMap[address, bool])
+fee: public(constant(uint256)) = 0  # 1 == 0.01 %
+
```

Comment:
could we add some sort of a 'MAX_LOAN_AMOUNT' value here, which is a constant of 3 million?